bin/take-dump.ts

```diff
@@ -3,13 +3,25 @@ import * as fs from 'fs'
 import * as path from 'path'
 import * as mkdirp from 'mkdirp'
 
+
+const env = process.env
+const CHAIN_ID = env.CHAIN_ID || '420'
```

Comment:
Can we make this environment var mandatory? Think it would save us a lot of headache to force everyone to explicitly specify their chainID when generating a dump.

Reply:
resolved, think it's ok after all